lightwood/mixers/helpers/transformer.py

```diff
@@ -14,10 +14,11 @@ def transform(self, sample):
         output_vector = []
 
         for input_feature in self.input_features:
-            sub_vector = sample['input_features'][input_feature].tolist()
-            input_vector.extend(sub_vector)
-            if input_feature not in self.feature_len_map:
-                self.feature_len_map[input_feature] = len(sub_vector)
+            if 'previous_' not in input_feature:  # TODO: fragile check for name
+                sub_vector = sample['input_features'][input_feature].tolist()
+                input_vector.extend(sub_vector)
+                if input_feature not in self.feature_len_map:
+                    self.feature_len_map[input_feature] = len(sub_vector)
```

Comment:
Why do we need to ignore the `previous_` features here ?

This seem like an unreliable way to do this, since we might want to jointly encode many more columns in the future.

Is it not easier to simply support "empty" encoders in the datasource, where a particular column is encoded to an empty list ?